library/core/src/mem/mod.rs

```diff
@@ -4,6 +4,7 @@
 //! types, initializing and manipulating memory.
 
 #![stable(feature = "rust1", since = "1.0.0")]
+#![cfg_attr(not(bootstrap), allow(unused-unsafe))]
```

Comment:
This is because stage1 requires functions that call intrinsics to have `unsafe` blocks, but stage0 does not and thus triggers the `unused_unsafe` lint (which is denied in rust-lang/rust).